File: nvim/lua/plugins/temp.go
package main 

import "fmt"

func fib(n int64) int64 {
  if(n<3){
    return 1
  }
  return fib(n-1) + fib(n-2)
}

func main(){
  fmt.Println("Hello, world")
  fmt.Println(fib(50));
}

File: nvim/lua/plugins/temp.js
function  fib(n){
  if (n<3){
    return 1;
  }
  else{
    return fib(n-1) + fib(n-2);
  }
}

console.log(fib(5));

async function main(a,b){
  return a ** b;
}

console.log(5,9)
console.log(fib())
